server/presenters/document.js

```diff
@@ -62,8 +66,14 @@ export default async function present(document: Document, options: ?Options) {
     pinned: undefined,
     collectionId: undefined,
     parentDocumentId: undefined,
+    lastViewedAt: undefined,
   };
 
+  if (!!userId) {
```

Comment:
When we query a document we already pass the `userId` along to the `findByPk` method in all of the api endpoints, for single documents we can ensure that the view is loaded in scope at that time and avoid an additional query in the presenter.

https://github.com/outline/outline/blob/develop/server/models/Document.js#L226-L228